libs/generic-view/feature/src/lib/generic-toasts.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React, { FunctionComponent, useMemo } from "react"
+import { useSelector } from "react-redux"
+import { ReduxRootState } from "Core/__deprecated__/renderer/store"
+import RecursiveLayout from "./recursive-layout"
+import { selectViewConfig } from "generic-view/store"
+import { createSelector } from "reselect"
+import { isEmpty } from "lodash"
+import styled from "styled-components"
+
+const selectToastsToRender = createSelector(selectViewConfig, (config) => {
+  return Object.entries(config || {})
+    .filter(([, { component }]) => ["toast"].includes(component))
+    .map(([key]) => key)
+})
+
+interface Props {
+  viewKey: string
+}
+
+export const GenericToasts: FunctionComponent<Props> = ({ viewKey }) => {
+  const toastsToRender = useSelector((state: ReduxRootState) =>
+    selectToastsToRender(state, { viewKey })
+  )
+  const toastsToRenderDependency = JSON.stringify(toastsToRender)
```

Comment:
Why is this here: const toastsToRenderDependency = JSON.stringify(toastsToRender)? Is it to work around the issue of array reference changes?

Reply:
We can leave this topic for the next sync

Reply:
Sure, let's discuss it later.